python/tests/kernel/test_run_kernel.py

```diff
@@ -916,6 +916,47 @@ def simple(numQubits: int) -> int:
     assert 'Invalid number of arguments passed to run:1 expected 0' in repr(e)
 
 
+def test_modify_struct():
+
+    @dataclass
+    class MyClass:
```

Comment:
Could the bool field come first so we also test for the values stored in the return object with correct alignment?